### ISSUE TITLE
[Bug]: Module 'angular-auth-oidc-client' has no exported member 'DefaultLocalStorageService'

### ISSUE THREAD

**Issue by daze99** [damienbod/angular-auth-oidc-client#1712]
### Version

15.0.3

### Please provide the exception or error you saw

```true
I am following the documentation (https://angular-auth-oidc-client.com/docs/documentation/configuration#using-localstorage-instead-of-default-sessionstorage) to use localstorage instead of sessionstorage.

Apparently the library has no exported member 'DefaultLocalStorageService'.
```

**Comment by timdeschryver** [comment]
I think this is not released yet, but the docs already mention it.

**Comment by damienbod** [comment]
released in 15.0.4
